### DISCUSSION TITLE
...continue routing

### DISCUSSION THREAD

**Discussion by petht** [organicmaps/organicmaps#5963] [Q&A]
Hi,
I could not solve the problem: when routing (bike) I interrupted the tour with "stop" (german: anhalten). Where/How can I find the button "continue"? Thanks for help.

**Comment by biodranik** [comment]
Stop cancels the navigation and resets the built route. What is your use case to pause it and continue? Why do you need to do that explicitly?

**Comment by petht** [comment]
E.g.: I would like to make a detour to a unplanned distant location for a meal break, or turn into a store something off the route, or visit a POI off the route etc. and after that come back to the planned tour.

If I set up the tour with several stops (Zwischenstops), everything has gone lost and I have to set up the tour again.

ok, in the end it leads to the fact that a planned route cannot be saved in OM at the moment.

**Reply by auge8472** [reply]
> E.g.: I would like to make a detour to a unplanned distant location for a meal break, or turn into a store something off the route, or visit a POI off the route etc. and after that come back to the planned tour.

Simply take your break but do not stop the routing in the app. It might look a bit disturbing (in iOS the top bar states the app to be still active, don't know how it looks in Android) but that shouldn't be a problem in itself.

**Comment by hb0nd** [comment]
i suppose this is sort of linked to https://github.com/organicmaps/organicmaps/issues/1785, in that there could be a 'pause' button, which saves the route state & returns to the normal map overview, where there is a button to resume or cancel that routing
